File: ios/stereoCamera/stereoCamera/Comm/Commands/SendDisconnect.swift
//
//  SendDisconnect.swift
//  stereoCamera
//
//  Created by hallmarklabs on 7/9/18.
//  Copyright © 2018 cody. All rights reserved.
//

import Foundation

class SendDisconnect : Command
{
    override init()
    {
        super.init()
        cmdtype = CommandTypes.RECEIVE_DISCONNECT
        expectsResponse = false
    }
}

File: android/app/src/main/java/com/munger/stereocamera/ip/command/commands/SendDisconnect.java
package com.munger.stereocamera.ip.command.commands;

import com.munger.stereocamera.ip.command.Command;

public class SendDisconnect extends Command
{
	public SendDisconnect()
	{
		super();
		cmdtype = Type.RECEIVE_DISCONNECT;
		expectsResponse = false;
	}
}
